.github/workflows/docker.yml

```diff
@@ -11,30 +11,32 @@ jobs:
   build-and-push:
     runs-on: ubuntu-latest
     steps:
```

Comment:
The new version of this is cribbed from the [tigerbeetle build step](https://github.com/coilhq/tigerbeetle/blob/main/.github/workflows/docker.yml#L24), which already builds for multiple architectures.